packages/client/components/SideBarStartMeetingButton.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {useLocation} from 'react-router'
+import useRouter from '~/hooks/useRouter'
+import getTeamIdFromPathname from '~/utils/getTeamIdFromPathname'
+import {BezierCurve} from '../types/constEnums'
+import FlatPrimaryButton from './FlatPrimaryButton'
+import Icon from './Icon'
+
+const Button = styled(FlatPrimaryButton)<{isOpen: boolean}>(({isOpen}) => ({
+  height: 50,
```

Comment:
+1 Any reason you increased the size to 50 instead of 48?

Reply:
No, I was having difficulty in centering the + icon and setting 50 instead of 48 resolved that, will try one more time if it is getting too big.
WDYT?